web/html/index.html

```diff
@@ -19,7 +19,155 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="../css/iframe.css">
+    <script>
+      if(exports){
+        var _tempExports = exports;
```

Comment:
This needs a code comment to warn about our temporary reservation of _tempExports.